assets/i18n/en_US.json

```diff
@@ -1,6 +1,7 @@
 {
     "okButton": "OK",
     "cancelButton": "Cancel",
+    "dismissButton": "Dismiss",
```

Comment:
Change this to `Cancel` instead.

Reply:
🔧 Resolved @ 00e49dba0cdcc71e07edad417800a6e2d4f1a4de